x-pack/solutions/observability/plugins/observability/public/pages/alert_details/components/dashboard_actions.tsx

```diff
@@ -0,0 +1,135 @@
+    },
+  } = useKibana();
+
+  const { alertId } = useParams<AlertDetailsPathParams>();
+  const [isLoading, alertDetail] = useFetchAlertDetail(alertId);
+  const ruleId = alertDetail?.formatted.fields[ALERT_RULE_UUID];
+  const { rule } = useFetchRule({
+    ruleId,
+  });
+
+  const [isActionsPopoverOpen, setIsActionsPopoverOpen] = useState(false);
+  const [dashboardToUnlink, setDashboardToUnlink] = useState<string | undefined>(undefined);
+  const [isDashboardUnlinking, setIsDashboardUnlinking] = useState(false);
+
+  const handleGoToDashboard = async () => {
+    const dashboardLocator = urlService.locators.get<DashboardLocatorParams>(DASHBOARD_APP_LOCATOR);
+    if (!dashboardLocator) {
+      return undefined;
+    }
+    dashboardLocator.navigate({ dashboardId });
+  };
+
+  const handleUnlinkDashboard = async () => {
+    setDashboardToUnlink({ dashboardId, ruleId });
+  };
+
+  const btn = (
+    <EuiButtonIcon
+      data-test-subj="o11ySloListItemButton"
+      aria-label={i18n.translate('xpack.observability.item.actions.button', {
+        defaultMessage: 'Actions',
+      })}
+      color="text"
+      display="empty"
+      iconType="boxesHorizontal"
+      size="s"
+      onClick={() => setIsActionsPopoverOpen(!isActionsPopoverOpen)}
+    />
+  );
+  if (isLoading || isDashboardUnlinking)
+    return (
+      <EuiFlexGroup data-test-subj="centerJustifiedSpinner" justifyContent="center">
+        <EuiFlexItem grow={false}>
+          <EuiLoadingSpinner size={'xl'} />
+        </EuiFlexItem>
+      </EuiFlexGroup>
+    );
+
+  return (
+    <>
+      <EuiPopover
+        anchorPosition="downLeft"
+        button={btn}
+        panelPaddingSize="none"
+        isOpen={isActionsPopoverOpen}
+      >
+        <EuiContextMenuPanel
+          items={[
+            <EuiContextMenuItem
+              key="viewDashboard"
+              icon="dashboardApp"
+              onClick={handleGoToDashboard}
+            >
+              {i18n.translate('xpack.observability.dashboardActions.viewDashboard', {
+                defaultMessage: 'Go to dashboard',
+              })}
+            </EuiContextMenuItem>,
+            <EuiContextMenuItem key="unlinkDashboard" icon="unlink" onClick={handleUnlinkDashboard}>
+              {i18n.translate('xpack.observability.dashboardActions.unlinkDashboard', {
+                defaultMessage: 'Remove from linked dashboards',
+              })}
+            </EuiContextMenuItem>,
+          ]}
+        />
+      </EuiPopover>
+      {dashboardToUnlink ? (
+        <UnlinkDashboardConfirmationModal
+          rule={rule}
+          ruleId={ruleId}
+          dashboardId={dashboardId}
+          title={dashboardTitle}
+          onCancel={() => setDashboardToUnlink(undefined)}
+          onDeleted={() => {
+            setDashboardToUnlink(undefined);
+            setIsDashboardUnlinking(false);
+            // location.reload(); TODO how do I reload the page with related dashboards tab selected?
```

Comment:
When I unlink a dashboard, I want to reload the page, so that it shows the updated dashboard list. How do I do this?

Reply:
Actions have been removed in this [commit](https://github.com/elastic/kibana/pull/219019/commits/651d8890625951fcca4cde93f5a20eecbbe588ee), so this comment is not relevant anymore.